### PR TITLE
Fix and add some French translations

### PR DESCRIPTION
**Pull request by YaFou**
Fixes mistakes like uppercase letters or missing translations.

> I invite all French people to contribute to this pull request.

### CODE REVIEW

File: core/assets/bundles/bundle_fr.properties [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 credits = Crédits
-contributors = Traducteurs et contributeurs
-discord = Rejoignez le Discord de Mindustry
-link.discord.description = Le discord officiel de Mindustry!
+contributors = Traducteurs et Contributeurs
+discord = Rejoignez le Discord de Mindustry !
+link.discord.description = Le Discord officiel de Mindustry!
 link.reddit.description = Le subreddit de Mindustry
 link.github.description = Code source du jeu
 link.changelog.description = Liste des mises a jour
```
Comment (YaFou) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
link.changelog.description = Liste des mises à jour
```

File: fastlane/metadata/android/fr-FR/full_description.txt [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 - Systèmes de transport d'énergie, liquides et objets
-- 19 différent types de drones, méchas et vaisseaux
+- 19 différents types de drones, méchas et vaisseaux
 - 120+ blocs technologiques à maîtriser
 - 75+ différents blocs environnementaux
-- Multijoueur cross-plateforme via réseau local or serveurs dédiés
-- Règles de jeu personnalisables: Changez le coût des structures, les stats des ennemis, les ressources de départ, fréquence des vagues et plus
-- Un éditeur puissant dotés d'outils pour générer aléatoirement des minéraux, le terrain, des décorations et appliquer une symétrie de terrain.
+- Multijoueur multiplateforme via un réseau local ou des serveurs dédiés
+- Règles de jeu personnalisables : Changez le coût des structures, les statistiques des ennemis, les ressources de départ, les fréquence des vagues et bien plus
```
Comment (Paulao17) [inline fastlane/metadata/android/fr-FR/full_description.txt]:
```suggestion
- Règles de jeu personnalisables : Changez le coût des structures, les statistiques des ennemis, les ressources de départ, la fréquence des vagues et bien plus
```

File: core/assets/bundles/bundle_fr.properties [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 link.dev-builds.description = Versions instables du jeu
 link.trello.description = Trello officiel pour les ajouts futurs
-link.itch.io.description = Page itch.io avec lien de téléchargement pour PC
+link.itch.io.description = Page itch.io avec les téléchargements pour PC
 link.google-play.description = Google Play Store
 link.f-droid.description = Catalogue F-Droid
 link.wiki.description = Le wiki officiel de Mindustry
-linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse papier.
+linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse-papiers.
```
Comment (Paulao17) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
linkfail = Erreur lors de l'ouverture du lien !\nL'URL a été copiée dans votre presse-papier.
```

File: fastlane/metadata/android/fr-FR/full_description.txt (hunk 2 above)
Comment (YaFou) [reply fastlane/metadata/android/fr-FR/full_description.txt]: Thank you for your review @Paulao17!

File: core/assets/bundles/bundle_fr.properties (hunk 3 above)
Comment (Paulao17) [reply core/assets/bundles/bundle_fr.properties]: According to [wikipedia](https://fr.wikipedia.org/wiki/Presse-papier_(informatique)), presse-papiers is the older writing.

File: core/assets/bundles/bundle_fr.properties [hunk 4]
```diff
@@ -26,222 +26,226 @@ load.image = Images
...
-schematic.importfile = Importer un fichier
-schematic.browseworkshop = Consulter le workshop
-schematic.copy = Copier au presse-papier
-schematic.copy.import = Importer du presse-papier
-schematic.shareworkshop = Partager sur le workshop
+schematic.exportfile = Exporter le Fichier
+schematic.importfile = Importer un Fichier
+schematic.browseworkshop = Consulter le Workshop
+schematic.copy = Copier dans le presse-papiers
+schematic.copy.import = Importer du presse-papiers
```
Comment (Paulao17) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
schematic.copy.import = Importer du presse-papier
```
Same here

**Comment by Anuken** [comment]
Is this PR stale?

**Comment by YaFou** [comment]
> 
> 
> Is this PR stale?

I am working on other pull requests and yes this pull request is stale. I will reopen it when I will be not busy.

**Pull request closed without merging**